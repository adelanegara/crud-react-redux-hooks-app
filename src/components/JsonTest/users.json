[
  {
    "albumId": 1,
    "id": 1,
    "title": "testing1",
    "url": "https://via.placeholder.com/600/92c952",
    "thumbnailUrl": "https://via.placeholder.com/150/92c952"
  },
  {
    "albumId": 1,
    "id": 2,
    "title": "testing2",
    "url": "https://via.placeholder.com/600/771796",
    "thumbnailUrl": "https://via.placeholder.com/150/771796"
  },
  {
    "albumId": 1,
    "id": 3,
    "title": "testing3",
    "url": "https://via.placeholder.com/600/24f355",
    "thumbnailUrl": "https://via.placeholder.com/150/24f355"
  },
  {
    "albumId": 1,
    "id": 4,
    "title": "testing4",
    "url": "https://via.placeholder.com/600/d32776",
    "thumbnailUrl": "https://via.placeholder.com/150/d32776"
  }
]
