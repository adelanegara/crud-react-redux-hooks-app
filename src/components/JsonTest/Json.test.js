import React from "react";
import users from "./users.json";

test("Welcome text test", () => {
  expect(users).toMatchSnapshot();
  expect(users).toHaveLength(4);
  expect(users.map((users1) => users1.title)).toEqual([
    "testing1",
    "testing2",
    "testing3",
    "testing4",
  ]);
});
