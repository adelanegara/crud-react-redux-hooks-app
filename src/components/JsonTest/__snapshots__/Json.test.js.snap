// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome text test 1`] = `
Array [
  Object {
    "albumId": 1,
    "id": 1,
    "thumbnailUrl": "https://via.placeholder.com/150/92c952",
    "title": "testing1",
    "url": "https://via.placeholder.com/600/92c952",
  },
  Object {
    "albumId": 1,
    "id": 2,
    "thumbnailUrl": "https://via.placeholder.com/150/771796",
    "title": "testing2",
    "url": "https://via.placeholder.com/600/771796",
  },
  Object {
    "albumId": 1,
    "id": 3,
    "thumbnailUrl": "https://via.placeholder.com/150/24f355",
    "title": "testing3",
    "url": "https://via.placeholder.com/600/24f355",
  },
  Object {
    "albumId": 1,
    "id": 4,
    "thumbnailUrl": "https://via.placeholder.com/150/d32776",
    "title": "testing4",
    "url": "https://via.placeholder.com/600/d32776",
  },
]
`;
